cli/flox/src/commands/list.rs

```diff
@@ -228,13 +237,18 @@ impl List {
                     ",
                         formatted_pname = formatted_pname.as_deref().unwrap_or(""),
                         description = description.as_deref().unwrap_or("N/A"),
-                        priority = DEFAULT_PRIORITY,
                         version = version.as_deref().unwrap_or("N/A"),
                         formatted_licenses = formatted_licenses.as_deref().unwrap_or("License: N/A"),
                         unfree = unfree.map(|u|u.to_string()).as_deref().unwrap_or("N/A"),
                         broken = broken.map(|b|b.to_string()).as_deref().unwrap_or("N/A"),
                     }
                 },
+                PackageToList::StorePath(locked_package_store_path) => formatdoc! {"
+                    {install_id}: {store_path}
+                    ",
+                    install_id = locked_package_store_path.install_id,
+                    store_path = locked_package_store_path.store_path
+                },
```

Comment:
(nit) Should we include the priority here? Should the path be nested underneath? It looks a bit odd compared to the others:
```
python3.12-yq: /nix/store/x2hwfhcs7mcd3sslvkvraw32l3wjksw2-python3.12-yq-3.4.3

hello: (hello)
  Description: Program that produces a familiar, friendly greeting
  Path:     hello
  Priority: 5
  Version:  2.12.1
  License:  GPL-3.0-or-later
  Unfree:   false
  Broken:   false

flox:
  Description:     N/A
  Locked URL:      github:flox/flox/e80f57cdaf5a70caed5dba9bba767cb2d6d2ce01
  Flake attribute: packages.aarch64-darwin.default
  Priority:        5
  Version:         1.3.5-ge80f57c
  License: N/A
  Unfree:          false
  Broken:          false
```